lms/djangoapps/verify_student/services.py

```diff
@@ -57,23 +58,18 @@ class IDVerificationService(object):
     """
 
     @classmethod
-    def user_is_verified(cls, user, earliest_allowed_date=None):
+    def user_is_verified(cls, user, deadline=None):
         """
         Return whether or not a user has satisfactorily proved their identity.
         Depending on the policy, this can expire after some period of time, so
         a user might have to renew periodically.
-
-        This will check for the user's *initial* verification.
         """
-        filter_kwargs = {
-            'user': user,
-            'status': 'approved',
-            'created_at__gte': (earliest_allowed_date or earliest_allowed_verification_date())
-        }
-
-        return (SoftwareSecurePhotoVerification.objects.filter(**filter_kwargs).exists() or
-                SSOVerification.objects.filter(**filter_kwargs).exists() or
-                ManualVerification.objects.filter(**filter_kwargs).exists())
+        expiration_datetime = cls.get_expiration_datetime(user, ['approved'])
+        if expiration_datetime:
+            if deadline:
```

Comment:
#NIT why not default set deadline to `now()` if it is `None`? It saves an `if` statement

Reply:
I couldn't find an instance where that extra parameter is actually used, so I just removed it.